### DISCUSSION TITLE
LOXILB WEBSITE ERROR 404

### DISCUSSION THREAD

**Discussion by jamroks** [loxilb-io/loxilb#648] [Q&A] [answered]
When trying to access the website,  we get the 404 error page , specially on blog page https://www.loxilb.io/not-found

**Comment by TrekkieCoder** [comment, marked as answer]
Hi @jamroks Can you please double check if access is via VPN ??

**Reply by jamroks** [reply]
Yes i was accessing via vpn , but it's working now that i deactivated vpn . thanks

**Reply by jamroks** [reply]
May i ask the question why is accessing the website blog via vpn problematic ?

**Reply by UltraInstinct14** [reply]
It is due to some security add-ons from webhosting provider(wix) to prevent web crawlers (behind vpn) to prevent overloading the servers and ddos.

**Reply by jamroks** [reply]
okay i understand, that is rather a little bit anoying, especially when traveling like i do, and accessing the blog from public wifi in airport or other public space like coworking office space, since i usually use vpn on those public wifi for obvious reasons.

**Reply by UltraInstinct14** [reply]
Duly noted. Will try to solve it !!

**Reply by jamroks** [reply]
thank you

**Comment by UltraInstinct14** [comment]
Working for me here 

![loxilb-Apr22-snap](https://github.com/loxilb-io/loxilb/assets/75648333/877de6eb-3e9b-4cda-b788-707779028e9a)
